tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -1656,7 +1660,7 @@ uint32_t SPIRVEmitter::doCXXMemberCallExpr(const CXXMemberCallExpr *expr) {
         return processBufferTextureLoad(object, location, constOffset,
                                         varOffset);
       }
-      llvm_unreachable("Load() is not implemented for the given object type.");
+      emitError("Load() is not implemented for the given object type.");
```

Comment:
`break` needed after this otherwise this will fall through?

Reply:
Oops, sorry i missed that. Good catch ;) Will return 0 since there's no return value otherwise.